packages/amplify_datastore/example/lib/models/AllScalarTypesListCustomType.dart

```diff
@@ -0,0 +1,383 @@
+/*
+* Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License").
+* You may not use this file except in compliance with the License.
+* A copy of the License is located at
+*
+*  http://aws.amazon.com/apache2.0
+*
+* or in the "license" file accompanying this file. This file is distributed
+* on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+* express or implied. See the License for the specific language governing
+* permissions and limitations under the License.
+*/
+
+// ignore_for_file: public_member_api_docs
+
+import 'ModelProvider.dart';
+import 'package:amplify_datastore_plugin_interface/amplify_datastore_plugin_interface.dart';
+import 'package:collection/collection.dart';
+import 'package:flutter/foundation.dart';
+
+/** This is an auto generated class representing the AllScalarTypesListCustomType type in your schema. */
+@immutable
+class AllScalarTypesListCustomType {
```

Comment:
Can you add the new models to https://github.com/aws-amplify/amplify-flutter/blob/main/packages/amplify_datastore/example/tool/schema.graphql?

That schema is used in the provisioning script for DataStore integ tests. It helps to add them there so that the model files can be re-generated. I also think it will be more important once we have integ tests that sync to the cloud.

Reply:
Oops I think I forgot to commit the change of schema.graphql 😅

Reply:
Added the missing changes.